### ISSUE TITLE
OutgoingMessage.prototype._headers is deprecated

### ISSUE THREAD

**Issue by balexandre** [Moesif/moesif-express#32]
using Windows 10 2004 on Node 14.5.0

```bash
GET /v2/registrations 200 (text/csv; charset=utf-8 32439bytes) - 119.977 ms
(node:23388) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
    at logEvent (E:\BitaBiz\api-registrations\node_modules\moesif-express\lib\index.js:421:9)
    at ServerResponse.res.end (E:\BitaBiz\api-registrations\node_modules\moesif-express\lib\index.js:283:11)
    at ServerResponse.send (E:\BitaBiz\api-registrations\node_modules\express\lib\response.js:221:10)
```

your code [at line 421](https://github.com/Moesif/moesif-express/blob/master/lib/index.js#L421)

**Comment by PauloGoncalvesBH** [comment]
This problem is also affecting me

**Comment by dgilling** [comment]
Hi @balexandre @PauloGoncalvesBH sorry for the delays responding to this, but our team is on it. Will let you know hopefully by this weekend.

**Comment by dgilling** [comment]
Hi, we have tested a fix in version 2.9.19. Let me know if you're still having issues.
